modules/qtwidgets/rawdatareaderdialogqt.h

```diff
@@ -74,9 +80,11 @@ class IVW_MODULE_QTWIDGETS_API RawDataReaderDialogQt : public VolumeDataReaderDi
     QLineEdit* spaceY_;
     QLineEdit* spaceZ_;
 
+    /*
     QSpinBox* timeSteps_;
     QSpinBox* headerOffset_;
```

Comment:
Offset is super important for a raw reader! Why do we remove this?

Reply:
Because there was only the widget thing, the value was never used anywhere... I left the code in there so we can add support for it

Reply:
Understood. I see it now.

core\io\rawvolumereader.cpp, Line 129 uses an offset of "0u" and not the value from the dialog. It should be easy to fix.

Shall I create a commit on this branch that fixes this?

Reply:
Sure